src/styles/main.scss

```diff
@@ -1 +1,59 @@
-// put code here
+body {
+  margin: 0;
+}
+
+.watch {
+  display: flex;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: $watch-size;
+  height: $watch-size;
+  margin: $watch-position 0 0 $watch-position;
+  border: 10px solid #000;
```

Comment:
color '#000' is duplicated, so please move all duplicated styles to the variables